### ISSUE TITLE
[bug] Can't build Android app using Tauri v2.0.0-rc.7

### ISSUE THREAD

**Issue by akkadaya** [tauri-apps/tauri#10760]
### Describe the bug

Error while building Android app

```log
Info Forwarding port 1420 with adbkage
adb: error: listener 'tcp:1420' not found
1420
error: failed to select a version for the requirement `hyper-rustls = "^0.27.0"`
candidate versions found which didn't match: 0.26.0, 0.25.0, 0.24.2, ...
location searched: crates.io index
required by package `reqwest v0.12.7`
    ... which satisfies dependency `reqwest = "^0.12"` (locked to 0.12.7) of package `tauri v2.0.0-rc.6`
    ... which satisfies dependency `tauri = "^2.0.0-rc"` (locked to 2.0.0-rc.6) of package `tauri-app v0.1.0 (/Users/xxx.xxx/Projects/tauri-app/src-tauri)`
if you are looking for the prerelease package it needs to be specified explicitly
    hyper-rustls = { version = "0.19.0-alpha.3" }
    Error `Failed to run `cargo build`: command ["cargo", "build", "--package", "tauri-app", "--manifest-path", "/Users/xxx.xxx/Projects/tauri-app/src-tauri/Cargo.toml", "--target", "aarch64-linux-android", "--features", "tauri/rustls-tls", "--lib", "--no-default-features"] exited with code 101
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
w: file:///Users/xxx.xxx/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-2.0.0-rc.6/mobile/android/src/main/java/app/tauri/plugin/PluginMethodData.kt:11:23 Parameter 'methodDecorator' is never used

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:rustBuildArm64Debug'.
> Process 'command 'yarn'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 19s
    Error Failed to assemble APK: command ["/Users/xxx.xxx/Projects/tauri-app/src-tauri/gen/android/gradlew", "--project-dir", "/Users/xxx.xxx/Projects/tauri-app/src-tauri/gen/android"] exited with code 1: command ["/Users/xxx.xxx/Projects/tauri-app/src-tauri/gen/android/gradlew", "--project-dir", "/Users/xxx.xxx/Projects/tauri-app/src-tauri/gen/android"] exited with code 1
error Command failed with exit code 1.
```

### Reproduction

```sh
yarn create tauri-app --rc
cd tauri-app
yarn
yarn tauri android init
yarn tauri android dev
```

### Expected behavior

build successfully

### Full `tauri info` output

```text
[✔] Environment
    - OS: Mac OS 14.6.1 X64
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.80.1 (3f5fd8dd4 2024-08-06)
    ✔ cargo: 1.80.1 (376290515 2024-07-16)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-aarch64-apple-darwin (environment override by RUSTUP_TOOLCHAIN)
    - node: 22.7.0
    - yarn: 1.22.22
    - npm: 10.8.2
    - bun: 1.1.26

[-] Packages
    - tauri [RUST]: 2.0.0-rc.6
    - tauri-build [RUST]: 2.0.0-rc.6
    - wry [RUST]: 0.42.0
    - tao [RUST]: 0.29.1
    - tauri-cli [RUST]: 2.0.0-rc.7
    - @tauri-apps/api [NPM]: 2.0.0-rc.3
    - @tauri-apps/cli [NPM]: 2.0.0-rc.7

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - framework: React
    - bundler: Vite
```

**Comment by akkadaya** [comment]
Fixed in Tauri v2.0.0-rc.8

**Comment by matejkoncal** [comment]
I have still this problem. Please help me.
